db/migrate/20200220173340_create_bpldc_authorities.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+class CreateBpldcAuthorities < ActiveRecord::Migration[5.1]
+  def change
+    create_table :bpldc_authorities do |t|
+      t.string :name
+      t.string :code, null: false
```

Comment:
maybe add a unique index on `code` but keep in mind this will be case sensitive so possibly we should force code values to be lowercase on create. thoughts?

Reply:
We have some values that are non-lowercase (`resourceTypes`) and non-alpha (`iso639-2`), so let's hold off on more model validation. Will add the unique index though.